bigflow/cli.py

```diff
@@ -268,7 +268,7 @@ def _parse_args(project_name: Optional[str], args) -> Namespace:
 
     _create_codegen_parser(subparsers)
 
-    return parser.parse_args(args)
+    return parser.parse_known_args(args)[0]
```

Comment:
WDYT about making it more obvious that the goal here is to return only bigflow args without extra args?
```suggestion
    bigflow_only_args = parser.parse_known_args(args)[0]
    return bigflow_only_args
```

Reply:
Can we test it somehow?

Reply:
I'll add some tests once we decide we're good with the change at all :)